sdk/core-rest/core-client/src/getClient.ts

```diff
@@ -0,0 +1,140 @@
+import { isTokenCredential, KeyCredential, TokenCredential } from "@azure/core-auth";
+import { PipelineOptions } from "@azure/core-rest-pipeline";
+import { createDefaultPipeline } from "./clientHelpers";
+import { HttpResponse } from "./common";
+import { RequestParameters } from "./pathClientTypes";
+import { sendRequest } from "./sendRequest";
+import { buildRequestUrl } from "./urlHelpers";
+
+/**
+ * Type to use with pathUnchecked, overrides the body type to any to allow flexibility
+ */
+export type PathUncheckedResponse = HttpResponse & { body: any };
```

Comment:
I originally had it as unknown, however users will be forced to cast the body to a specific type. In general this would be good, but the use case of PathUnchecked is for calling arbitrary paths, for which we haven't had generated types this would be an "advanced" scenario where we just assume users know what they are doing.

The common case `path` has body as unknown by default and overloads the type depending on the path being used